pkg/apis/rbac/helpers.go

```diff
@@ -234,7 +234,7 @@ func (r *PolicyRuleBuilder) Rule() (PolicyRule, error) {
 				}
 			}
 			if len(illegalVerbs) > 0 {
-				return PolicyRule{}, fmt.Errorf("verbs %v do not have names available: %#v", illegalVerbs, r.PolicyRule)
+				return PolicyRule{}, fmt.Errorf("verbs %v do not support resource name: %#v", illegalVerbs, r.PolicyRule)
```

Comment:
verbs don't really "support" resource name... the current message is accurate, for these verbs, the resource name is not generally available